### DISCUSSION TITLE
Is there a way to disable inline?

### DISCUSSION THREAD

**Discussion by ShravanSunder** [aguingand/tiptap-markdown#21] [General]
@aguingand  I'd like to disable inline in 
```ts
 insertContentAt: (range, content, options) => (props) => {
                return commands.insertContentAt(
                    range,
                    props.editor.storage.markdown.parser.parse(content, {
                        inline: true,
                    }),
                    options
                )(props);
            },
```

The reason is i'm streaming markdown into the editor, and inline is causing wonky issues with my stream inserts.  while insertcontent works really well.  How does inline work and what is it suppposed to do?

## perhaps its not the right solution for my issue 🤔
i need a way to stream two lines for example that are list-items without impacting the depth

if i stream (using insertContent) the following line by line (with \n preceeding the new line)
```
- first list item
```
```

- second list item
```

they would get rendered as
```
- first list item
  - second list item
```

Any ideas on how to solve this?

This indent issue seems to happen if i stream a paragraph after a block quote for example

**Comment by aguingand** [comment]
I don't think this is related to `inline`, this is one drawback of markdown you can't render a single `<li>...</li>`. In your example 
```markdown

- second list item
```
will be rendered to 
```html
<ul>
  <li>
    second list item
  </li>
</ul>
```
So it'll be appended as sublist. 
You can try to insert it as HTML or using ˋsetContent` to replace the whole document
